Model/ReachCurrency.php

```diff
@@ -106,7 +106,7 @@ public function getCurrencyRates($currency, $toCurrencies = null)
     {
         if (in_array($this->state->getAreaCode(), ['frontend','webapi_rest']) && $this->reachHelper->isReachCurrencyEnabled()) {
             if (!$this->_storeManager->getStore()->isCountryApplicable()) {
-                return parent::getRate($toCurrency);
+                return parent::getRate($toCurrency); //where is this $toCurrency coming from???
```

Comment:
I'm honestly confused about what this entire section is doing, $toCurrency would be null and I think it's a typo from a copy/paste from the other functions that have $toCurrency as the second parameter since this function has $toCurrencies instead of $toCurrency

Reply:
I recognize that this is overriding the getCurrencyRates method from the Magento Currency parent class but I don't know why. I don't know if these comments are helpful haha but I'm thinking out loud

Reply:
I flagged this/added the comment - so that it is looked at or get fixed at some point.